src/components/MyListItem/MyListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { User } from '../../models/user';
```

Comment:
nit: these types of imports with a lot of `../` are generally annoying since it gets in the way of moving the file through the project, you generally want something like ` @/models/user` 

there is an eslint rule for that
https://github.com/andrepapoti/react-test/pull/1

Reply:
added `baseUrl` to tsconfig